src/main/java/org/usfirst/frc4904/auton/JustMove.java

```diff
@@ -0,0 +1,11 @@
+package org.usfirst.frc4904.auton;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisMoveDistance;
+
+class JustMove extends AutonRoutine {
+
+    public JustMove() {
+        this.andThen(new ChassisMoveDistance(RobotMap.Component.chassis, 10, null)); // TODO: what motioncontroller?
```

Comment:
I may be mistaken, but doesn't andThen require you to already have some commands in the group?

Reply:
What do you do instead to add the first command?